src/views/preview/project-view.jsx

```diff
@@ -205,14 +209,29 @@ class Preview extends React.Component {
                                 }
                             });
                         }
+                        const sprites = projectData[0].targets.length - 1; // don't count stage
+                        const scripts = projectData[0].targets
+                            .map(target =>
+                                Object.values(target.blocks)
+                                    .filter(block => block.topLevel).length
```

Comment:
I checked with the mod team and top-level blocks is fine with them even though it's a change from 2.0